### ISSUE TITLE
🐛 Bug Report: Generate an administrator token to be able to manage the rbac plugin from within the code.

### ISSUE THREAD

**Issue by axie10** [backstage/backstage#28288]
### 📜 Description

hello very good morning. How can I have an administrator token in my backstage application (that does not expire) to be able to call endpoints that have backstage or the different plugins like RBAC to be able to do actions in cdoigo that do not require the token of the user who is logged in. Is there any way to generate one and configure it? Thank you very much!

### 👍 Expected behavior

I am trying to manage the rbac plugin from the code but in the put, post calls to manage roles and give permissions to users I always get that I am not authorized with error 401. I have generated a static token and through the configuration that backstage gives and I can assign the restrictions: create, delete, update and read but it doesn't work. Is there any way to generate an admin token that allows you to do all this? Thanks!

### 👎 Actual Behavior with Screenshots

I can't manage the rbac plugin (for all the APIs it exposes, only for GET requests) from the code. 

### 👟 Reproduction steps

Try to manage the RBAC plugin via code.

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

**Comment by axie10** [comment]
@freben Here is my problem in a little more detail

**Comment by pguppy** [comment]
Not giving any accessRestrictions or permissions on the static token should give it 
`unlimited access to all functionality of all plugins`
[base on the docs](https://backstage.io/docs/auth/service-to-service-auth#access-restrictions)

**Comment by axie10** [comment]
@pguppy  At first I did not give any permissions or any plugin and yet it keeps giving error. Then I made requests to that api, the one to update roles from postman and the error that comes out (which is more detailed) is this:

![image](https://github.com/user-attachments/assets/cd14f8f3-cbd6-4e07-99d0-c8e240bca9fb)

In fact in the plugin there are several conditionals where it specifies that the token has to be user type with the “allow”. Really the sense that it has when putting the restriction of the token type user is that when it makes the updtae of the role, in the metadata it saves who has modified it. The objective would be to be able to create an admin and user type token. The three validations that it makes inside the plugin:

1. At first it only accepts token type “user” and “service”:
![image](https://github.com/user-attachments/assets/a2e3dd48-5f9b-4b59-8366-68e493c18ef2)


2. If the token is type “service” and you are going to do an action other than “pluginRbaccommon.PolicyEntityReadPermission” it does not let you pass which is the error I get in postman:
![image](https://github.com/user-attachments/assets/4ae02aef-b63d-4a2b-a43d-fa8ce1579b7d)


3. Once inside the update role function, it tells you again to allow “user”. And as I was saying before in the metadata it puts you the user that has updated the role (as it can be observed in this one I have it modified so that it accepts that type of token and modified by remains undefined):
![image](https://github.com/user-attachments/assets/10ff92d0-f41e-4167-9036-9f2ac318c2ac)

**Comment by freben** [comment]
Yeah the plugin maker has explicitly chosen to only let humans administer things.

**Comment by benjdlambert** [comment]
I also want to call out movement for #27680 that's going to help you generate tokens for communicating with Backstage based on a user, but again this might not fulfil you're entire use case here as these tokens will also expire. 

In general, it's really not good practice to be issuing tokens that don't expire, especially with full admin controls to different plugins. It's a security hole that we'd really advise adopters not to pursuit.

**Comment by axie10** [comment]
The goal is to be able to manage many users dynamically without having to be registering one by one by hand in the plugin, to be able to assign users to existing roles automatically.

Regarding the comment @freben , how could I contact the creator of this plugin? Thank you!

**Comment by freben** [comment]
Check git blame for the lines in question, that's probably easiest

**Comment by axie10** [comment]
@04kash Hi, I don't know how to contact you, I'm naming you here because this is where I described the issue. I have seen that last month you did the commit of the file policies-rest-api.ts and I have this doubt in question, which I describe above. I don't know if there can be a solution for this. Thank you very much in advance!

**Comment by 04kash** [comment]
Hi @axie10, the rbac plugin is currently being maintained by @AndrienkoAleksandr, @divyanshiGupta,@christoph-jerolimov and @PatAKnight (as mentioned [here](https://github.com/backstage/community-plugins/blob/main/.github/CODEOWNERS#L45)), and might be the best to assist with your question!

**Comment by axie10** [comment]
Hi, could you give me a hand with this case that is happening to me with the rbac plugin or how I could apply if you see feasible to modify it or if you can not do anything? To have something clear. Thank you very much in advance! @christoph-jerolimov @PatAKnight @divyanshiGupta @AndrienkoAleksandr
